src/app.js

```diff
@@ -71,9 +87,9 @@ process.on('unhandledRejection', (reason) => {
   throw new ErrorHandler(reason);
 });
 
-process.on('uncaughtException', (error) => {
-  logger.error(`Uncaught Exception: ${500} - ${error.message}, Stack: ${error.stack}`);
-  process.kill(process.pid, 'SIGTERM');
-});
+// process.on('uncaughtException', (error) => {
+//   logger.error(`Uncaught Exception: ${500} - ${error.message}, Stack: ${error.stack}`);
+//   process.kill(process.pid, 'SIGTERM');
+// });
```

Comment:
Why was this commented out? I think it's supposed to help handle uncaught exceptions, logs the error and gracefully shuts down the server.

Reply:
It was a mistake. Now it's resolved.